src/components/ColorPicker.component.tsx

```diff
@@ -12,13 +13,15 @@ export const ColorPicker = ({
   hideHEX = false,
   hideRGB = false,
   hideHSV = false,
+  alpha = false,
   dark = false,
 }: ColorPickerProps): JSX.Element => (
   <div className={`rcp ${dark ? "rcp-dark" : "rcp-light"}`} style={{ width }}>
     <Saturation width={width} height={height} color={color} onChange={onChange} />
     <div className="rcp-body">
       <Hue width={width - 40} color={color} onChange={onChange} />
-      <Fields color={color} hideHEX={hideHEX} hideRGB={hideRGB} hideHSV={hideHSV} onChange={onChange} />
+      {alpha ? <Alpha width={width - 40} color={color} onChange={onChange} /> : null}
```

Comment:
Is it just me or the `&&` a cleaner way to do optional rendering so that `null` is not needed.

What I mean is this
```tsx
{alpha && <Alpha width={width - 40} color={color} onChange={onChange} />}
```

Reply:
> Is it just me or the `&&` a cleaner way to do optional rendering so that `null` is not needed.
> 
> What I mean is this
> 
> ```tsx
> {alpha && <Alpha width={width - 40} color={color} onChange={onChange} />}
> ```

Oh. I didn't notice. Thank you ❤️